apps/server/src/app/reporting/requests.resolver.ts

```diff
@@ -29,14 +29,16 @@ export class RequestReportsResolver {
     @Args("data") data: GetRequestsInput,
     @CurrentUser() user: RequestUser
   ): Promise<RequestReportResult> {
+    let organizationId;
     try {
       const project = await this.projectsService.getProjectById(data.projectId);
 
       if (!project) {
         throw new NotFoundException();
       }
 
-      isOrgMemberOrThrow(user, project.organizationId);
+      organizationId = project.organizationId;
+      isOrgMemberOrThrow(user, organizationId);
     } catch (error) {
       this.logger.error(error, "Error getting projects");
       throw new InternalServerErrorException();
```

Comment:
Should not throw a `NotFoundException` inside another try-catch block. Please split it into multiple try-catch blocks.

Reply:
This looks like a code I've written, but it actually isn't. I only added `organizationId = project.organizationId;`

Due to the multi approach concept (the two commits in this PR) I'd like to keep the commit history clean until we make a decision. Will be fixed right after